src/components/ErrorMessagePanel/ErrorMessagePanel.js

```diff
@@ -0,0 +1,24 @@
+import { Typography, makeStyles } from "@material-ui/core"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: 0,
+    color: theme.palette.error.main,
+  },
+}))
+
+const ErrorMessagePanel = ({ errors }) => {
+  const classes = useStyles()
+
+  return (
+    <Typography
+      variant="caption"
+      display="block"
+      className={classes.root}
+    >
+      {!!errors && errors.message}
```

Comment:
You should not be rendering anything at all if there is no error message, consider making use of early returns